app/main.py

```diff
@@ -1 +1,57 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    base_skills = ["JavaScript", "HTML", "CSS"]
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = FrontendDeveloper.base_skills
```

Comment:
You'll have an issue here. Each FrontendDeveloper instance will have skills that refers to global list of skills "base_skills". And when you make self.learn_skill('new skill'), it'll add a new skill to self.skills, but self.skills is base_skills, thus each class instance will learn this skills and this is incorrect